fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -827,6 +828,20 @@ alter_stmt ::=
     {:
         RESULT = new AlterRoutineLoadStmt(jobLabel, jobProperties, datasourceProperties);
     :}
+    | KW_ALTER KW_TABLE table_name:tbl KW_SET KW_STATS LPAREN key_value_map:map RPAREN
```

Comment:
Could you please add this new statement in #6277 ?
I think they should be under `Database Administration Statements`(same as MySQL)

Reply:
DONE